bucket_map/src/index_entry.rs

```diff
@@ -153,4 +164,23 @@ mod tests {
         let mut index = IndexEntry::new(Pubkey::new_unique());
         index.set_storage_offset(too_big);
     }
+
+    #[test]
+    fn test_data_bucket_from_num_slots() {
```

Comment:
Can you update the test to include 0 as well?

Reply:
done.